notebooks/03-query/09-functions-case.ipynb

```diff
@@ -0,0 +1,499 @@
+    "SELECT\n",
+    "  TIME_PARSE(\"timestamp\") AS \"__time\",\n",
+    "  \"browser\",\n",
+    "  \"event_type\",\n",
+    "  \"event_subtype\",\n",
+    "  \"loaded_image\",\n",
+    "  \"session_length\",\n",
+    "  \"session\"\n",
+    "FROM \"ext\"\n",
+    "PARTITIONED BY DAY\n",
+    "'''\n",
+    "\n",
+    "display.run_task(sql)\n",
+    "sql_client.wait_until_ready('example-koalas-conditions')\n",
+    "display.table('example-koalas-conditions')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1b6c9b88-837d-4c80-a28d-36184ba63355",
+   "metadata": {},
+   "source": [
+    "## Understanding the dataset\n",
+    "\n",
+    "The koalas to the max dataset records events from [Koalas to the Max](https://www.koalastothemax.com). Before you start, visit the site to see how it operates.\n",
+    "\n",
+    "- When a user first opens an image, a \"GoodLoad\" type event is posted.\n",
+    "- As you uncover more of the image, \"PercentClear\" type events are posted. The amount of the image that has been cleared is recorded as a value in `event_subtype`.\n",
+    "- An image is comprised of layers – \"LayerClear\" events are issued every-so-often, and the layer cleared is recorded in the `event_subtype`.\n",
+    "- When a user clears an entire image, a \"LayerClear\" event is recorded without a value in `event_subtype`.\n",
+    "\n",
+    "Run the following cell to see how this journey shows up in the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7e2eac10-bd0b-4fd3-84f3-b49a846a85d3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "SELECT \"__time\",\n",
+    "  \"event_type\",\n",
+    "  \"event_subtype\",\n",
+    "  \"loaded_image\"\n",
+    "FROM \"example-koalas-conditions\"\n",
+    "WHERE session = 'S89403399'\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ba25bdec-c11a-4423-bb5c-b38f2d09944b",
+   "metadata": {},
+   "source": [
+    "## CASE functions at query time\n",
+    "\n",
+    "The `CASE` function provides if-then-else behaviour in two forms.\n",
+    "\n",
+    "- The simplest form switches between different outputs based on the value of a dimension.\n",
+    "- The searched form allows for more complex comparison operations.\n",
+    "\n",
+    "This notebook uses the SQL versions of `CASE`. Native (JSON-based) versions [are also available](https://druid.apache.org/docs/27.0.0/querying/math-expr#general-functions), allowing them to be used in streaming ingestion, for example.\n",
```

Comment:
Suggest applying the hyperlink to "Native (JSON-based) versions" rather than "are also available"